R/dataset_vignette.R

```diff
@@ -0,0 +1,74 @@
+#' Output a dataset in a vignette in the admiral format
+#'
+#' Output a dataset in a vignette with the pre-specified format
+#' By default, the variables specified in `display_vars`
+#'
+#' @param dataset Dataset to output in the vignette
+#'
+#' @param display_vars Variables selected to demonstrate the outcome of the derivation.
+#'
+#' Permitted Values: list of variables
+#'
+#' Default is NULL
+#'
+#' If `display_vars` is not NULL, only the selected variables are visible in the vignette while the
+#' other variables are hidden. They can be made visible by clicking the`Choose the columns to
+#'  display` button.
+#'
+#' @param filter Filter condition
+#'
+#' The specified condition is applied to the dataset before it is displayed.
+#'
+#' Permitted Values: a condition
+#'
+#' @keywords vignette
```

Comment:
Either you should add a "Vignette" section to the "Developer Utilities" section or you should use one of the existing keywords. Otherwise the function is not listed on the "Reference" page.

Reply:
Ok, i have used the `dev_utility` keyword